Python/pytime.c

```diff
@@ -35,6 +35,59 @@ error_time_t_overflow(void)
                     "timestamp out of range for platform time_t");
 }
 
+static void
+_PyTime_overflow(void)
+{
+    PyErr_SetString(PyExc_OverflowError,
+                    "timestamp too large to convert to C _PyTime_t");
+}
+
+#if defined(__APPLE__) || defined(MS_WINDOWS)
+/* Euclidean algorithm to compute the greatest common divisor of a and b. */
+static _PyTime_t
+_PyTime_GCD(_PyTime_t a, _PyTime_t b)
+{
+    _PyTime_t x, y;
```

Comment:
Why not reuse a and b as in common implementation of Euclid algorithm? This would slightly simplify the code. In any case you need a one temporary variable.

Reply:
Ok, will do once I will rebase this PR.